contracts/src/libraries/FROST.sol

```diff
@@ -262,54 +262,76 @@ library FROST {
     // ============================================================
 
     /**
-     * @notice Hashes input to a field element for binding factors.
-     * @param input The input bytes.
+     * @notice h1: Hashes input to a field element for calculating binding factors (rho).
+     * @param input The input bytes, which include group public key, message hash, and commitments hash.
```

Comment:
The input is just input bytes to the hash function. How the hash function is used is specific to the outer algorithm, and should not be included in the natspec here.


```suggestion
     * @param input The input bytes.
```